### PR TITLE
45723 Implement logging configuration

### PR DESCRIPTION
**Pull request by ganpat-tech**
PR contains code for logging configuration  :-
UKHO EventHub logger
Azure AppInsights logger
File based logger (only for local development)

### CODE REVIEW

File: NVDSuppressions.xml [hunk 1]
```diff
@@ -9,4 +9,129 @@
...
+    </notes>
+    <cve>CVE-2021-34485</cve>
+  </suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Serilog.Sinks.File:4.1.0
+      8117-softmagic.c in file before 5.21 does not properly limit recursion, which allows remote attackers to cause a denial of service (CPU consumption or crash) via unspecified vectors.
+      9652-The mconvert function in softmagic.c in file before 5.21, as used in the Fileinfo component in PHP before 5.4.37, 5.5.x before 5.5.21, and 5.6.x before 5.6.5, does not properly handle a certain string-length field during a copy of a truncated version of a Pascal string, which might allow remote attackers to cause a denial of service (out-of-bounds memory access and application crash) via a crafted file.
+    This is a false positive. These two CVEs references a C library softmagic.c which is not used by ESS
```
Comment (usha13529) [inline NVDSuppressions.xml:103]:
```suggestion
    This is a false positive. These two CVEs references a C library softmagic.c which is not used by MSI
```

File: NVDSuppressions.xml [hunk 2]
```diff
@@ -9,4 +9,129 @@
...
+   A vulnerability exists in Async through 3.2.1 (fixed in 3.2.2) , which could let a malicious user obtain privileges via the mapValues() method.
+   ]]></notes>
+   <cve>CVE-2021-43138</cve>
+</suppress>
+  <suppress>
+    <notes>
+      <![CDATA[
+    file name: Serilog.Extensions.Logging.File:2.0.0
+      cdf_read_property_info in cdf.c in file through 5.37 does not restrict the number of CDF_VECTOR elements, which allows a heap-based buffer overflow (4-byte out-of-bounds write)
+    This is a false positive in cdf.c which is not used by ESS
```
Comment (usha13529) [inline NVDSuppressions.xml:132]:
```suggestion
    This is a false positive in cdf.c which is not used by MSI
```

**Review by usha13529 — CHANGES_REQUESTED** [review]
Minor comments to address please.

File: NVDSuppressions.xml (hunk 1 above)
Comment (ganpat-tech) [reply NVDSuppressions.xml]: Thanks for highlighting, done change.

File: NVDSuppressions.xml (hunk 2 above)
Comment (ganpat-tech) [reply NVDSuppressions.xml]: done change.

File: NVDSuppressions.xml [hunk 3]
```diff
@@ -9,4 +9,129 @@
...
+    ]]>
+    </notes>
+    <cve>CVE-2014-9653</cve>
+  </suppress>
+  <suppress>
+   <notes>
+     <![CDATA[
+   file name: Serilog.Sinks.Async.dll
+   A vulnerability exists in Async through 3.2.1 (fixed in 3.2.2) , which could let a malicious user obtain privileges via the mapValues() method.
+   ]]></notes>
```
Comment (barrie-cooper) [inline NVDSuppressions.xml:124]:
```suggestion
   This is a false positive as the vulnerability exists in a NPM Javascript package named 'Async' and not the nuget .NET package named 'Serilog.Sinks.Async'.
   ]]></notes>
```

Comment (ganpat-tech) [reply NVDSuppressions.xml]: Thanks for highlighting, done change.

**Review by usha13529 — APPROVED** [review]
Thanks for the changes @ganpat-tech. LGTM.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @ganpat-tech

**Pull request merged**